tasks.py

```diff
@@ -136,7 +136,7 @@ def _detect_space(branch=None, yes=False):
 
 DEPLOY_RULES = (
     ('prod', _detect_prod),
-    ('stage', lambda _, branch: branch == 'master'),
+    ('stage', lambda _, branch: branch == 'release'),
```

Comment:
Actually, thinking about this a bit, release branches aren't the only way for code to get to master--we also have hotfixes to deal with. We could add hotfixes to the `lambda` here, but I'm not sure if that would capture the workflow correctly. For example:

* I submit a hotfix from my own fork
* The build passes, but since I'm not pushing to 18F, no deploy is triggered
* The fix gets merged into master with a release tag and is deployed to production